config/testdata/config/invalid/transport-invalid-queue.toml

```diff
@@ -0,0 +1,17 @@
+[transport]
```

Comment:
If we simplify how we do configurations then we will not need extensive (or any) tests just for parsing the config file.